packages/scaffolding-cli/templates/src/ssr/environment-configuration/environment-variables.js

```diff
@@ -13,4 +13,14 @@ module.exports = {
     AUTH0_DOMAIN: process.env.AUTH0_DOMAIN,
     AUTH0_CLIENT_SECRET: process.env.AUTH0_CLIENT_SECRET,
     AUTH0_CALLBACK_URL: `${process.env.APP_BASE_URL}:${process.env.PORT}/callback`,
+
+    NEXT_PUBLIC_CONTENTFUL_SPACE_ID:
+        process.env.NEXT_PUBLIC_CONTENTFUL_SPACE_ID || "veka2or2977j",
+    NEXT_PUBLIC_CONTENTFUL_ACCESS_TOKEN:
+        process.env.NEXT_PUBLIC_CONTENTFUL_ACCESS_TOKEN || "",
+    NEXT_PUBLIC_CONTENTFUL_PREVIEW_ACCESS_TOKEN:
+        process.env.NEXT_PUBLIC_CONTENTFUL_PREVIEW_ACCESS_TOKEN ||
```

Comment:
Is this a public access token?

Reply:
Preview token is only read only so I believe it is fine to leave it there.